### ISSUE TITLE
[FEATURE_REQUEST] <title>

### ISSUE THREAD

**Issue by nickm324** [Lissy93/dashy#597]
### Describe the solution you'd like

Would like an additional option called DISCARD added to the Opening Methods that would discard the response result completely instead of opening it in a default New Tab.

### Priority

Medium (Would be very useful)

### Is this something you would be keen to implement

No

**Comment by Lissy93** [comment]
Hiya,
I'm not sure what you mean by _"discard the response result completely instead of opening it in a default New Tab"_.
Could you elaborate? Thanks

**Comment by nickm324** [comment]
Hi! I really love the interface and all its features and was working on something that would expand on it by being able to have links that would run applications on the workstation.

I have written a small proxy type application that sits in the system tray and accepts HTTP request, and when a request and would send some commands via query string parameters. The app would then have permissions to open the executable. The only issue at the moment is that when clicking the link, it works perfectly in opening the application I specified but it also opens a web browser tab that is basically blank or shows no response.

I was wondering if it would be possible to just call the link and then discard or ignore any response, which I am assuming you are doing something to open the response in the modal or workspace.
I can easily send a response but really don't need a new tab or anything to open after the command is sent to proxy app.
Below is an example of a URL that would open a specific file in notepad by clicking one of the links in Dashy.

http://localhost:2222/?EXEPATH=notepad.exe&ARGS=c:\temp\logs\Prox20220410.log&DEBUG=true

**Comment by Lissy93** [comment]
Makes sense, I think this is the same as Action Buttons mentioned in #383, which is already on the [Roadmap](https://github.com/Lissy93/dashy/discussions/405).
That will allow you to trigger an action by calling a web hook or hitting a HTTP endpoint.

But it's not quite as simple as just adding an extra option to the opening method. It would need to be a function that makes a HTTP request from the frontend to the endpoint (and you would need to manage that endpoint and CORS and stuff yourself)

**Comment by nickm324** [comment]
#383 is along the same line but more for running server-side commands to control things inside the container that is hosting Dashy. 

My idea allows Dashy to be used to perform actions on the client computer. 

Since a user would have to manually install the proxy and configure it and formulate the URLs to execute whatever application I see it as a very low security risk. Yet it would be simple to get up and running.

 Overall, this provides a complete solution to use Dashy as an all-in-one Local/Remote dashboard.

I may fork the code and take a look at the current process and see if I can come up with a solution.
The only real requirement is for Dashy to shoot off the request as normal and then forget about it after that point.
